clang/test/dpct/device001.cu

```diff
@@ -197,4 +199,47 @@ cudaDeviceProp* getcudaDevicePropPtr() {
 void test4() {
   //CHECK:int a1 = getcudaDevicePropPtr()->get_max_compute_units();
   int a1 = getcudaDevicePropPtr()->multiProcessorCount;
-}
\ No newline at end of file
+}
+
+__device__ void test5() {
+  cudaDeviceProp *pDeviceProp = nullptr;
```

Comment:
Suggest adding check for cudaDeviceProp.

Reply:
I've updated the lit test as suggested. Thanks